### ISSUE TITLE
Documentation: example MaterialsBuilder script

### ISSUE THREAD

**Issue by tylerhuntington222** [materialsproject/emmet#91]
In the example MaterialsBuilder script under 'Running a Builder' in the root directory's README  I'm trying to understand why the `MongoStore` constructor receives the `collection_name="materials"` kwarg for instantiating the `tasks_store` object, and `collection_name="tasks"` for instantiating `materials_store`.  Should these parameters be swapped? Or am I misunderstanding the conceptual/functional role of each Store instance?

**Comment by mkhorton** [comment]
Yes, they should probably be swapped. The readme was written very early on before much of the builders themselves were written and could probably be updated. Certainly, some simpler examples could be added now (we try to use the MapBuilder base class as much as possible, but I don’t think that’s in the readme).

What’s your use case?

**Comment by dwinston** [comment]
@mkhorton Tyler's "use case" is helping us. He's one of the MP part-timers starting this month. :)

**Comment by tylerhuntington222** [comment]
@mkhorton thanks for the clarification! The script makes a lot more sense with those parameters switched. And, yes, right now just trying to orient myself to the MP codebase so I can help out over the next few months.
